x-pack/plugins/ml/public/application/contexts/kibana/use_current_theme.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import useObservable from 'react-use/lib/useObservable';
+import { of } from 'rxjs';
+import { useMemo } from 'react';
+import { euiDarkVars as euiThemeDark, euiLightVars as euiThemeLight } from '@kbn/ui-theme';
+import { useMlKibana } from './kibana_context';
+
+const themeDefault = { darkMode: false };
+
+/**
+ * Indicates if the currently applied theme is either dark or light.
+ * @return {boolean} - Returns true if the currently applied theme is dark.
+ */
+export function useCurrentKibanaTheme() {
```

Comment:
I think this could have a clearer name, maybe `useIsDarkMode ` or `useIsDarkTheme` something like that.
The current name sounds like it's going to return the current theme rather than a boolean.